plone-and-python-community.rst

```diff
@@ -1830,13 +1829,13 @@ Plone Roadmap 2020
 
 .. container:: overlay centered
 
-    **Onboard the Plone Community and join the journey**
+    **Join the Plone Community and join the journey**
 
     See you at
 
-    * Plone Open Garden 2017 - Sorrento - Italiy - 18-22. April 2017
-    * PloneConf 2017 Barcelona - Catalonia - 16-22.  October 2017
-    * or any Sprint or Event, ...
+    * Plone Open Garden 2017 - Sorrento - Italy - April 18-22, 2017
+    * PloneConf 2017 Barcelona - Catalonia - October 16-22, 2017
+    * or any sprint or event, ... plone.org/events
```

Comment:
Ah, just looked at http://maurits.vanrees.org/weblog/archive/2016/10/lightning-talks-thursday?searchterm=slovenia "Nejc Zupan: Dragonsprint. This is a Pyramid sprint in my home town of Ljubjana, Slovenia. 5 to 9 December this year. It would be great to have Plone people there too." so it isn't for Plone.